src/cow-react/modules/swap/services/common/types.ts

```diff
@@ -38,4 +38,6 @@ export type SwapFlowContext = BaseFlowContext & {
 export type EthFlowContext = BaseFlowContext & {
   contract: CoWSwapEthFlow
   addTransaction: ReturnType<typeof useTransactionAdder>
+  existsInFlightOrderId: (orderId: string) => boolean
```

Comment:
Nitpick. The words order is weird. Also, it's an action rather to check the existence rather than the boolean.
What about:

```suggestion
  checkInFlightOrderIdExists: (orderId: string) => boolean
```

Reply:
Done